app/dao/fact_notification_status_dao.py

```diff
@@ -313,6 +319,86 @@ def fetch_notification_status_for_service_for_today_and_7_previous_days(service_
     ).all()
 
 
+def fetch_daily_notification_status_for_service(service_id, by_template=False):
```

Comment:
this is a new function? I don't see where it is called in this PR?

Reply:
Ah my bad, at first I was going to make a new method so I wouldnt affect other things, but I thought better of that.  Let me clean that up.